code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -30,8 +30,8 @@ Contains:
 	return
 
 /obj/item/clothing/suit/space/hardsuit/deathsquad
-	name = "deathsquad suit"
-	desc = "A heavily armored, advanced space suit that protects against most forms of damage."
+	name = "MK.III SWAT Suit"
+	desc = "A prototype designed to replace the ageing MK.II SWAT suit. Based on the streamlined MK.II model, the traditional ceramic and graphene plate construction was replaced with plasteel, allowing superior armor against most threats. There's room for some kind of energy projection device on the back."

```

Comment:
You are not editing the correct hardsuit. See https://github.com/tgstation/tgstation/pull/21933/files#diff-6f8cb79de28d012195aecce429bafe5eR697 for the current shielded units currently in use.